src/lib/wasm/MultipassStore.ts

```diff
@@ -0,0 +1,67 @@
+import { get, writable, type Writable } from "svelte/store"
+import init, * as wasm from "warp-wasm"
+import { WarpStore } from "./WarpStore"
+
+
+class MultipassStore {
+    private multipassWritable: Writable<wasm.MultiPassBox | null>
+    private identityProfile: Writable<wasm.IdentityProfile | null> = writable(null)
```

Comment:
I wouldnt advise storing `IdentityProfile` but instead just store `Identity` from it since `IdentityProfile` has the potential of storing the mnemonic phrase depending on if it was supplied or not during creation.